database/migrations/2022_10_20_155323_create_tasks_table.php

```diff
@@ -16,7 +16,9 @@ public function up()
         Schema::create('tasks', function (Blueprint $table) {
             $table->increments('id');
             $table->string('name', 50);
-            $table->timestamps();
+            $table->string('date', 50);
+            $table->string('time', 50);
+            // $table->timestamps();
```

Comment:
you are creating a new table
Add date and time fields to existing table

Reply:
solved.